client/src/services/genai.ts

```diff
@@ -0,0 +1,769 @@
+/**
```

Comment:
We wanna version these in git?

Reply:
Yes, this resolves communication mismatches with Steve